packages/fxa-auth-client/lib/utils.ts

```diff
@@ -29,6 +29,14 @@ export function uint8ToBase64Url(array: Uint8Array) {
     .replace(/\//g, '_');
 }
 
+export function base64UrlToUint8(value: string): Uint8Array {
+  const m = value.length % 4;
+  return Uint8Array.from(atob(
+   value.replace(/-/g, '+')
+    .replace(/_/g, '/')
```

Comment:
I don't think we need to use regex in these calls.

Reply:
When I removed this, I wasn't getting correct `recoveryData` generated from previous test examples :/

Reply:
Oh `replace`'s default behavior is to replace the first match. ☹️